### ISSUE TITLE
Synthetic models

### ISSUE THREAD

**Issue by xwasco** [danini/progressive-x#1]
Dear @danini, thanks for your awesome work :)

I am trying your code and I wonder if you can also add the functions to reproduce Figure 4 of your paper.

Thanks!

Best regards

**Comment by danini** [comment]
Dear @xwasco ,

Thanks :) I can, but I will most likely not have time to do that before ECCV deadline. After that, I certainly will.

Best,
Daniel

**Comment by xwasco** [comment]
It would be great if you can before ECCV :)

Thanks in any case!

All the best

Il dom 1 mar 2020, 12:53 Daniel Barath <notifications@github.com> ha
scritto:

**Comment by ThoMut** [comment]
Hello @danini 

I also would be interested in code reproducing Figure 4 of your paper.
I hope you find some time to add the functions.

greetings, Thomas

**Comment by xwasco** [comment]
Still interested too

best

Seba

**Comment by gabriel-h-ribeiro** [comment]
Hi, @danini!

Hope this message finds you well. I was reading your article and I was impressed by your algorithm. I noticed that you generated some examples of synthetic models (Figure 4), but I couldn't reproduce it with the code that is in the repository...

Could you provide me with the code snippet please? If possible, could you send me the link to the specific file on Github?

Thanks in advance for your help and willingness to share your knowledge with me. It will be of great help for my learning.

Yours sincerely.

**Comment by danini** [comment]
Yeah, I didn't add multi-line fitting to the repository. I will upload it later this week.

**Comment by Tigores** [comment]
Dear @danini ,

Looking forward to your multi-line fitting example!!!

**Comment by danini** [comment]
I added multi-line fitting here: https://github.com/danini/progressive-x/blob/master/examples/example_multi_lines.ipynb
I haven't performed any parameter tuning on line fitting problems, so the results probably are suboptimal.
